Fitbit.Portable/FitbitClientExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Fitbit.Models;
+
+namespace Fitbit.Api.Portable
+{
+    internal static class FitbitClientExtensions
```

Comment:
@mxa0079 like I said in the original PR I wasn't that sure on the name so happy to go with that :-)